test/GithubApi.Gist.test.js

```diff
@@ -0,0 +1,91 @@
+require('dotenv').config();
+
+const { StatusCodes } = require('http-status-codes');
+const { expect } = require('chai');
+const chai = require('chai');
+const axios = require('axios');
+chai.use(require('chai-subset'));
+
+const urlBase = 'https://api.github.com';
+const promise = '"This file should contain a promise"';
```

Comment:
delete `promise` const and use it directly

Reply:
Done